packages/asc-ui/src/components/Toggle/Toggle.tsx

```diff
@@ -1,160 +1,37 @@
-import {
-  ButtonHTMLAttributes,
-  ElementType,
-  FunctionComponent,
-  KeyboardEvent,
-  HTMLAttributes,
-  MouseEvent,
-  useCallback,
-  useEffect,
-  useRef,
-  useState,
-} from 'react'
-import ToggleStyle, { Props as ToggleStyleProps } from './ToggleStyle'
-import ownerDocument from '../../utils/ownerDocument'
-import usePassPropsToChildren from '../../utils/hooks/usePassPropsToChildren'
-import useActionOnEscape from '../../utils/hooks/useActionOnEscape'
-import ToggleButton, {
-  Props as ToggleButtonProps,
-} from '../Button/ToggleButton/ToggleButton'
-import BackDrop, { Props as BackDropProps } from '../BackDrop/BackDrop'
+import { forwardRef, HTMLAttributes, useImperativeHandle, useRef } from 'react'
+import RadioStyle, { Props, LabelStyle, WrapperStyle } from './ToggleStyle'
```

Comment:
Is it a convention to separate these styles into another file? Because this component is so small I'd prefer placing all the components in 1 file.

Reply:
are you sure about that? does Jon agree about this as well?

Reply:
I don't know if he's the keeper of an convention. My preference is move them into 1 file. IMO It improves readability.

Reply:
gedaan